server/integration-tests/calabash-test-suite/Gemfile.lock

```diff
@@ -3,7 +3,7 @@ GEM
   specs:
     awesome_print (1.8.0)
     builder (3.2.3)
-    calabash-android (0.9.3)
+    calabash-android (0.9.5)
```

Comment:
We should not have made this change.

I am not sure what the best solution is.

In the release branch, I will update the Gemfile to point to "calabash/calabash-android" master branch.

And I will remove the Gemfile.lock from the git index.